mainnet_contract/contracts/LiteCheckpoint.sol

```diff
@@ -0,0 +1,424 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.19;
+
+import {HeaderReader} from "./libraries/HeaderReader.sol";
+
+contract LiteCheckpoint {
+    struct HeaderInfo {
+        uint64 number;
+        uint64 roundNum;
+        int64 mainnetNum;
+    }
+
+    struct UnCommittedHeaderInfo {
+        uint64 sequence;
+        uint64 lastRoundNum;
+        uint64 lastNum;
+    }
+
+    struct BlockLite {
+        bytes32 blockHash;
+        uint64 number;
+    }
+
+    struct Validators {
+        address[] set;
+        int256 threshold;
+    }
+    mapping(bytes32 => bytes32) private unCommittedLastHash;
```

Comment:
what's the meaning of this? parent hash?

Reply:
> what's the meaning of this? parent hash?

https://github.com/XinFinOrg/XDC-Subnet/pull/75#discussion_r1296065760

Reply:
please add explanation of two hash in `mapping(bytes32 => bytes32)` to comment